### ISSUE TITLE
Issue with installation of Autowasp in Burp CE

### ISSUE THREAD

**Issue by kibelous** [GovTech-CSG/Autowasp#7]
Hello,

I faced with issue while installing the extension. The error is 

```
java.lang.NullPointerException: Cannot invoke "burp.IBurpCollaboratorClientContext.generatePayload(boolean)" because "this.extender.iBurpCollaboratorClientContext" is null
	at autowasp.logger.TrafficLogic.<init>(TrafficLogic.java:69)
	at autowasp.Autowasp.registerExtenderCallbacks(Autowasp.java:98)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at burp.amj.lambda$registerExtenderCallbacks$0(Unknown Source)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
```

How can I fix it? I've tried to install extension through BApp Store, compiling and adding compiled file and adding pre-compiled file. 

<img width="996" alt="image" src="https://user-images.githubusercontent.com/23276701/112723824-299c5e00-8f19-11eb-9a10-085ea7081652.png">

**Comment by kibelous** [comment]
My bad, I used incorrect version

**Comment by kibelous** [comment]
I got a new error in Burp Suite 1.7.13

```
java.lang.NullPointerException
	at autowasp.logger.TrafficLogic.<init>(TrafficLogic.java:69)
	at autowasp.Autowasp.registerExtenderCallbacks(Autowasp.java:98)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at burp.p1c.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)

```

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/23276701/112729371-6cb7fa80-8f34-11eb-93e7-54fce31d92a5.png">

**Comment by vay3t** [comment]
i have the same error in CE v2021.3.1

**Comment by kibelous** [comment]
I guess the extension doesn't work in Burp CE because my college checked in Pro version and everything is OK.

**Comment by vay3t** [comment]
but

![image](https://user-images.githubusercontent.com/6080369/112750293-38792400-8f9e-11eb-8971-5b594db7c055.png)

**Comment by imthomas93** [comment]
Hi all!

Thank you for leaving dropping letting us know about this issue. I have checked and we did was advised by PortSwigger to incorporate IBurpCollaboratorClientContext APIs before we publish this tool. Because we are using Burp Collaborator features which is a Pro only feature, Autowasp will not work in Burp Community issue for now.

I will follow up with PortSwigger to see if they would recommend any alternative workaround for this issue.
